### ISSUE TITLE
Explore alternate icon sets

### ISSUE THREAD

**Issue by melchoyce** [WordPress/twentyseventeen#65]
Right now I'm using Genericons with one custom icon, but it might be nice to have something stylistically different this year. We have a lot of flexibility here. Only requirement is that the icon set uses a GPL-compatible open source license.

The icons currently used in the theme:
- Menu (Hamburger) icon
- X
- Down arrow
- Left arrow
- Right arrow
- Down chevron
- Search
- Quote
- Category
- Tag (ideally a hash)
- Reply
- Social icons (ideal to have a wide variety here — could be good to grab a social-specific icon set for this, as long as it was built on the same grid)

**Comment by acosmin** [comment]
These icons would go really well with the current font-weight in your designs:

http://colebemis.com/feather/ - MIT License

and for social we can use FontAwesome. Combine them both with IcoMoon...

**Comment by melchoyce** [comment]
Thanks @acosmin! I think the outlined style could look really nice with the theme. It looks like it's missing some icons, but I could always try to extend it myself to cover the missing ones.

**Comment by tobeycodes** [comment]
The latest version of genericons is quite different and doesn't include social logos. I'd suggest using something else or combining it with automattic/social-logos

https://github.com/Automattic/Genericons/pull/98

**Comment by seraphyn** [comment]
What about icons purely made from css?
[CSS Icon set](http://cssicon.space/)

**Comment by josephfusco** [comment]
@seraphyn 

IMO Complex pure-CSS icons are not practical. They can be difficult to maintain and are often only possible with an un-semantic mess of markup. That icon set you linked, while an admirable project, is over 11,000 lines of CSS.

**Comment by melchoyce** [comment]
Thinking we should just go with Font Awesome. Nothing else had the same breadth of available options (especially for social icons) and/or compatible license.

Looks pretty okay:

![post_with_font_awesome_icons](https://cloud.githubusercontent.com/assets/2846578/19194644/489232ca-8c7c-11e6-8d8c-710355f58d7a.jpg)

**Comment by melchoyce** [comment]
Waiting on #206 to close.

**Comment by laurelfulford** [comment]
Fixed with #322.
